content/efcore/getting-started/installing-ef-core.md

```diff
@@ -1,28 +1,32 @@
 # Installing EF Core  
  
-In this lesson, you will learn how to install EF Core on your machine. You do not need to complete these steps to continue with the rest of the tutorial as the code examples in the other lessons are executable in-browser.
+In this lesson, you will learn how to install Entity Framework Core on your machine. The other lessons in this tutorial use code examples that are executable in-browser, so you do not need to complete the steps in this lesson to continue.
  
 ## Install .NET Core 
  
-If you haven't already, you will first need to install .NET Core on your machine. Go to the [.NET Core Page](https://www.microsoft.com/net/core) of the Microsoft website, and follow the installation instructions for your platform of choice. You can ensure .NET Core is installed by running the following command: 
+Once again, you _do not_ have to complete this lesson to go on to the other lessons. This lesson serves as a reference so you can get started with EF Core, and it is only necessary to complete the steps of this lesson if you want to follow along with the tutorial locally on your machine.
```

Comment:
This probably isn't as relevant since we are moving where Installing EF Core is. I'm getting rid of _all_ the warnings of "you don't need ta do this"

🌮